epel9.tpl

```diff
@@ -0,0 +1,60 @@
+config_opts['chroot_setup_cmd'] += " epel-rpm-macros"
```

Comment:
here?

Reply:
this would literally just write `dnf epel-rpm-macros`

Reply:
I think what's going on here is that the template is trying to add that package to the install command on the EL9 template. I removed that line in the EPEL template and moved the `epel-rpm-macros` package to be installed with the chroot setup cmd on the EL9 template. At this point, I'm just unsure whether I need to keep the `config_opts['chroot_setup_cmd'] = 'install @{% if mirrored %}buildsys-{% endif %}build'` line.